.github/workflows/sub_gradle_test_and_build.yml

```diff
@@ -47,7 +51,7 @@ jobs:
       - name: Gradle test and build. (unit tests, integration tests, end-2-end tests and build)
         env:
           run_docker: false
-        run: ./gradlew clean build --no-daemon --stacktrace
+        run: ./gradlew clean build --no-daemon --stacktrace -x spotlessApply -x spotlessCheck -x spotlessKotlin -x spotlessKotlinApply -x javadoc -x javadocJar -x sourcesJar
```

Comment:
I think we can skip `spotlessApply`, but not check. If a developer tries to submit unformatted code, we should be catching it in CI.

Reply:
@philipliu If `spotlessCheck` fails, does it fail the CI?

Reply:
Sorry, I missed this earlier, but we should keep `spotlessKotlin` too right?

Reply:
No problem at all. Thanks for catching these gotchas. I had removed `-x spotlessKotlin`.